include/diaspora.php

```diff
@@ -1170,6 +1228,8 @@ private function receive_comment($importer, $sender, $data, $xml) {
 
 		$datarray["body"] = diaspora2bb($text);
 
+		$datarray["body"] = self::replace_people_gui($datarray);
```

Comment:
I think it should be `$datarray["body"] = self::replace_people_gui($datarray["body"]);` here?